src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1

```diff
@@ -147,6 +152,36 @@ class SVTResourceResolver: AzSKRoot {
         }
     }
 
+    [void] GetResourceCount($projectName) {
+        [Hashtable] $projectData = @{
+            projectName = $projectName;
+            repositories = 0;
+            testplan = 0;
+            pipelines = 0;
```

Comment:
pipelines will give just build pipelines. we should also compute release pipelines.